northstar/src/api/model.rs

```diff
@@ -224,7 +226,7 @@ pub enum UmountResult {
 pub enum Response {
     Ok,
     Error(Error),
-    ContainerStats(Container, ContainerStats),
+    Inspect(Box<ContainerData>),
     Containers(Vec<ContainerData>),
```

Comment:
This clashes with `Inspect`. Think those two should be aligned because they carry the same result type.

either

```
Inspect(...),
InspectAll(...),
```

or

```
Container(...),
Containers(...),
```